nilearn/_utils/class_inspect.py

```diff
@@ -1,29 +1,25 @@
 """Small utilities to inspect classes."""
 
-from sklearn.utils.estimator_checks import (
-    check_decision_proba_consistency,
-    check_estimator_get_tags_default_keys,
-    check_estimators_partial_fit_n_features,
-    check_get_params_invariance,
-    check_non_transformer_estimators_n_iter,
-    check_set_params,
-)
 from sklearn.utils.estimator_checks import (
     check_estimator as sklearn_check_estimator,
 )
 
+# List of sklearn estimators checks that are valid
+# for all nilearn estimators.
 VALID_CHECKS = [
-    x.__name__
-    for x in [
-        check_estimator_get_tags_default_keys,
-        check_estimators_partial_fit_n_features,
-        check_non_transformer_estimators_n_iter,
-        check_decision_proba_consistency,
-        check_get_params_invariance,
-        check_set_params,
-    ]
+    "check_estimator_cloneable",
+    "check_estimator_get_tags_default_keys",
```

Comment:
```suggestion
    "check_valid_tag_types",
```
I think we need to update this as suggested.
This would fix the pre-release test failure, right?

Reply:
I think so!

Reply:
will fix